Doc/library/types.rst

```diff
@@ -86,6 +86,28 @@ Dynamic Type Creation
 
    :pep:`560` - Core support for typing module and generic types
 
+.. function:: get_orig_bases(tp, /)
+
+    Returns the objects in the bases list in the class's definition before
```

Comment:
We generally prefer the imperative mood for the first sentence of a docstring or the docs:

```suggestion
    Return the objects in the bases list in the class's definition before
```